### ISSUE TITLE
Update Datapack Compatibility to new versions of Minecraft

### ISSUE THREAD

**Issue by Redls07** [jtorleon-studios-team/bettervillages#114]
I am playing on 1.20.1, but I have problems with the Bountiful mod due to the lack of compatibility between mods

**Comment by vinnyg0621** [comment]
yeah when the coniferous forest village update comes out I'd like to see this as well
